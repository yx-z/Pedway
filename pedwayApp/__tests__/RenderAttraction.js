import RenderAttraction from '../components/RenderAttractions/RenderAttractions';
import testRenderer from 'react-test-renderer';
import React from 'react';
import PedwayMockData from '../mock_data/attractions';


test('Check if our RenderAttractions renders correctly', () => {
  const treeRendered = testRenderer.create(<RenderAttraction />).toJSON();
  expect(treeRendered).toMatchSnapshot();
});


test('test JSON helpers', ()=>{
  const treeRendered = testRenderer.create(<RenderAttraction
    JSONData={PedwayMockData}/>).toJSON();
  expect(treeRendered.length).toBe(16);
});
