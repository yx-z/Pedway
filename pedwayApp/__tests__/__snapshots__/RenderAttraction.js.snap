// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if our RenderAttractions renders correctly 1`] = `null`;
